filters/auth/oidc_test.go

```diff
@@ -825,6 +825,12 @@ func TestOIDCSetup(t *testing.T) {
 			}
 			bs := string(b)
 			t.Logf("Got body: %s", bs)
+
+			// make sure cookies have correct maxAge according to exp
+			for _, c := range client.Jar.Cookies(reqURL) {
+				maxAge := time.Duration(c.MaxAge * int(time.Second)).Round(time.Hour)
```

Comment:
Lets round to a smaller value, e.g. 1 minute.

The cast to int does not feel safe/canonical, I think it should be `time.Duration(c.MaxAge) * time.Second` as per https://pkg.go.dev/time#Duration
> To convert an integer number of units to a Duration, multiply:

    seconds := 10
    fmt.Print(time.Duration(seconds)*time.Second) // prints 10s

Reply:
I think we should rather check that MaxAge is within `[2h - delta(e.g. 1 minute), 2h]` interval instead of rounding.

Reply:
Yeah, that makes sense. What do you think about:

```suggestion
				maxAge := time.Duration(c.MaxAge) * time.Second
				assert.InDelta(t, 2*time.Hour, maxAge, float64(time.Minute))
```
(though here I don't really like the conversion to float64)

Reply:
InDelta is not really correct as it must not be above 2h, maybe:
```go
	maxAge := time.Duration(10) * time.Second
	if maxAge < 2*time.Hour-time.Minute || maxAge > 2*time.Hour {

	}
```